person.rb

```diff
@@ -5,14 +5,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] The `Student` class should have a method that serves as the 'belongs-to' side (a student belongs to a classroom).

Reply:
Well resolved 👍🏽